tests/foreman/cli/test_satellitesync.py

```diff
@@ -16,97 +16,117 @@
 
 :Upstream: No
 """
-import json
+import os.path
```

Comment:
why not pathlib's Path?

Reply:
Was not aware of pathlib so used os.path for path manipulation.  I can try to implement pathlib and see if that fits my needs for some of the path extraction.

Reply:
I'd consider this optional for the scope of this PR, but would also strongly recommend moving to using `PurePath` for your use-case in this module.

Reply:
It is optional, but recommended :D